README.md

```
@@ -9,7 +9,7 @@
 
 
 #### How  it works
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)